.github/workflows/e2e.yaml

```diff
@@ -15,11 +15,12 @@ env:
   # TODO: match BASEIMAGE with Makefile default (nonroot variant)
   BASEIMAGE: "gcr.io/distroless/static-debian11"
   KIND_CLUSTER_NAME: "kind"
+  KIND_VERSION: "v0.27.0"
 
 jobs:
   build:
     name: build
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
The Ubuntu-20.04 brownout takes place from 2025-02-01. For more details, see 
- https://github.com/actions/runner-images/issues/11101